build-rkt

```diff
@@ -0,0 +1,11 @@
+#!/bin/sh
+
+export SRC_DIR=`pwd`
+
+sudo rkt run \
+    --volume src-dir,kind=host,source=${SRC_DIR} \
+    --volume build-dir,kind=host,source=${SRC_DIR} \
+    --interactive \
```

Comment:
On Tue, Jul 19, 2016 at 5:50 PM, Stefan Junker <notifications@github.com>
wrote:

> he --interactive argument is not needed here, as the build process will
> not require human interaction. I hope I'm not wrong with the latter ;-)
>

probably correct. I'm indifferent. This is what was presented to me as how
rkt was built withing rkt. I did not question it; just passed it along

*--Richard Bucker *

*http://about.me/richard.bucker <http://about.me/richard.bucker>*